pipeline/extract/gog/extract_gog.py

```diff
@@ -163,7 +172,7 @@ def search_pages_last_day() -> list:
     load_dotenv()
     counter = 1
     games_from_page = req.get(
-        f'{ENV["GOG_BASE_URL"]}&page={counter}', timeout=5)
+        f'{ENV["GOG_BASE_URL"]}&page={counter}', timeout=1)
```

Comment:
Just curious as to why the timeout is lower?

Reply:
Good spot, this was a mistake!